scIB/metrics/__init__.py

```diff
@@ -0,0 +1,16 @@
+__all__ = [
+    "ari",
+    "cell_cycle",
+    "graph_connectivity",
+    "highly_variable_genes",
+    "isolated_labels",
+    "kbet",
+    "lisi",
```

Comment:
Do we distinguish `lisi` and `graph_lisi`?

Reply:
yes, I didn't change the function names. This is just the module name, the functions are available under `scIB.me.lisi.lisi()` and `scIB.me.lisi.lisi_graph()`, respectively. Additionally I also make them directly available via `scIB.me.lisi()` and `scIB.me.lisi_graph()`.

Reply:
OK!

Reply:
i think that's possible as well... at least for all functions that should be outward facing.

Reply:
Exactly, both `scIB.me.lisi.lisi()` and `scIB.me.lisi()` are possible